pkg/tsdb/elasticsearch/response_parser.go

```diff
@@ -374,6 +374,10 @@ func (rp *responseParser) processAggregationDocs(esAgg *simplejson.Json, aggDef
 
 				if len(otherMetrics) > 1 {
 					metricName += " " + metric.Field
+					if metric.Type == "bucket_script" {
+						//Use the formula in the column name
+						metricName, _ = metric.Settings.Get("script").String()
```

Comment:
```suggestion
						metricName = metric.Settings.MustString("script")
```

Reply:
Sorry, this was what it should have been.

```suggestion
						metricName = metric.Settings.Get("script").MustString("")
```